Fix travis builds

Building node from source fails:

```
  lone:compile:make stdout +639ms   g++ '-DV8_TARGET_ARCH_X64' '-DENABLE_DISASSEMBLER' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_STATIC' -I../deps/v8  -pthread -Wall -Wextra -Wno-unused-parameter -m64 -B/home/travis/build/mongodb-js/lone/test/.lone/node-v5.8.0/third_party/binutils/Linux_x64/Release/bin -fno-strict-aliasing -m64 -O3 -ffunction-sections -fdata-sections -fno-omit-frame-pointer -fdata-sections -ffunction-sections -O3 -fno-rtti -fno-exceptions -std=gnu++0x -MMD -MF /home/travis/build/mongodb-js/lone/test/.lone/node-v5.8.0/out/Release/.deps//home/travis/build/mongodb-js/lone/test/.lone/node-v5.8.0/out/Release/obj.target/v8_base/deps/v8/src/accessors.o.d.raw   -c -o /home/travis/build/mongodb-js/lone/test/.lone/node-v5.8.0/out/Release/obj.target/v8_base/deps/v8/src/accessors.o ../deps/v8/src/accessors.cc
  lone:compile:make stderr +126ms In file included from ../deps/v8/src/accessors.h:8:0,
                 from ../deps/v8/src/accessors.cc:5:
../deps/v8/include/v8.h:469:1: error: expected unqualified-id before ‘using’
  lone:compile:make stderr +2ms ../deps/v8/include/v8.h:856:1: error: expected unqualified-id before ‘using’
  lone:compile:make stderr +190ms In file included from ../deps/v8/src/accessors.h:11:0,
                 from ../deps/v8/src/accessors.cc:5:
../deps/v8/src/handles.h:76:7: error: template argument required for ‘class Handle’
../deps/v8/src/handles.h:76:20: error: expected initializer before ‘:’ token
  lone:compile:make stderr +0ms ../deps/v8/src/handles.h:147:7: error: template argument required for ‘class MaybeHandle’
../deps/v8/src/handles.h:147:25: error: invalid type in declaration before ‘{’ token
../deps/v8/src/handles.h:147:25: error: template declaration of ‘int v8::internal::final’
../deps/v8/src/handles.h:148:2: error: expected primary-expression before ‘public’
../deps/v8/src/handles.h:148:2: error: expected ‘}’ before ‘public’
../deps/v8/src/handles.h:150:25: error: expected class-name before ‘(’ token
../deps/v8/src/handles.h:155:41: error: ISO C++ forbids declaration of ‘MaybeHandle’ with no type [-fpermissive]
../deps/v8/src/handles.h:155:41: error: declaration of template ‘template<class S> int v8::internal::MaybeHandle(v8::internal::Handle<T>)’
../deps/v8/src/globals.h:415:29: error: conflicts with previous declaration ‘template<class T> struct v8::internal::MaybeHandle’
../deps/v8/src/globals.h:415:29: error: previous non-function declaration ‘template<class T> struct v8::internal::MaybeHandle’
../deps/v8/src/handles.h:155:41: error: conflicts with function declaration ‘template<class S> int v8::internal::MaybeHandle(v8::internal::Handle<T>)’
../deps/v8/src/handles.h: In function ‘int v8::internal::MaybeHandle(v8::internal::Handle<T>)’:
../deps/v8/src/handles.h:156:9: error: only constructors take member initializers
../deps/v8/src/handles.h:156:36: error: expected type-specifier before ‘T’
../deps/v8/src/handles.h:156:36: error: expected ‘>’ before ‘T’
../deps/v8/src/handles.h:156:36: error: expected ‘(’ before ‘T’
../deps/v8/src/handles.h:156:36: error: ‘T’ was not declared in this scope
../deps/v8/src/handles.h:156:39: error: expected primary-expression before ‘>’ token
../deps/v8/src/handles.h:206:26: error: expected ‘{’ before ‘;’ token
../deps/v8/src/handles.h:206:26: warning: no return statement in function returning non-void [-Wreturn-type]
../deps/v8/src/handles.h: At global scope:
../deps/v8/src/handles.h:211:3: error: ‘friend’ used outside of class
../deps/v8/src/handles.h:214:3: error: ‘friend’ used outside of class
../deps/v8/src/handles.h:218:29: error: ‘v8::hash_value’ declared as an ‘inline’ variable
../deps/v8/src/handles.h:218:29: warning: ‘always_inline’ attribute ignored [-Wattributes]
../deps/v8/src/handles.h:218:29: error: template declaration of ‘size_t v8::hash_value’
../deps/v8/src/handles.h:218:29: error: ‘MaybeHandle’ was not declared in this scope
../deps/v8/src/handles.h:218:29: note: suggested alternative:
../deps/v8/src/globals.h:415:29: note:   ‘v8::internal::MaybeHandle’
../deps/v8/src/handles.h:218:42: error: expected primary-expression before ‘>’ token
../deps/v8/src/handles.h:218:44: error: ‘maybe_handle’ was not declared in this scope
../deps/v8/src/handles.h:237:7: error: redefinition of ‘class v8::HandleScope’
../deps/v8/include/v8.h:873:17: error: previous definition of ‘class v8::HandleScope’
../deps/v8/src/handles.h:302:27: error: variable ‘v8::DeferredHandleScope v8::final’ has initializer but incomplete type
../deps/v8/src/handles.h:303:2: error: expected primary-expression before ‘public’
../deps/v8/src/handles.h:303:2: error: expected ‘}’ before ‘public’
../deps/v8/src/handles.h:303:2: error: expected ‘,’ or ‘;’ before ‘public’
../deps/v8/src/handles.h:308:3: error: ‘DeferredHandles’ does not name a type
../deps/v8/src/handles.h:309:25: error: expected constructor, destructor, or type conversion before ‘;’ token
../deps/v8/src/handles.h:311:2: error: expected unqualified-id before ‘private’
../deps/v8/src/handles.h:314:3: error: ‘HandleScopeImplementer’ does not name a type
../deps/v8/src/handles.h:321:3: error: ‘friend’ used outside of class
../deps/v8/src/handles.h:327:23: error: variable ‘SealHandleScope final’ has initializer but incomplete type
../deps/v8/src/handles.h:328:2: error: expected primary-expression before ‘public’
../deps/v8/src/handles.h:328:2: error: expected ‘}’ before ‘public’
../deps/v8/src/handles.h:328:2: error: expected ‘,’ or ‘;’ before ‘public’
../deps/v8/src/handles.h:331:20: error: declaration of ‘~SealHandleScope’ as non-member
../deps/v8/src/handles.h:340:1: error: expected declaration before ‘}’ token
  lone:compile:make stderr +20ms make[1]: *** [/home/travis/build/mongodb-js/lone/test/.lone/node-v5.8.0/out/Release/obj.target/v8_base/deps/v8/src/accessors.o] Error 1
  lone:compile:make stderr +1ms make: *** [node] Error 2
  lone:compile:make exit code +0ms 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,9 +1,15 @@
-sudo: false
+sudo: true
 language: node_js
 node_js:
   - 5
-env: DEBUG=lone*
+env: DEBUG=lone* CXX=g++-4.8 CI=1
 script: npm run-script ci
 cache:
   directories:
   - node_modules
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8

